liquid.py

```diff
@@ -128,7 +142,8 @@ def run(cmd):
 
 def run_fg(cmd, **kwargs):
     kwargs.setdefault('shell', True)
-    subprocess.check_call(cmd, **kwargs)
+    kwargs.setdefault('stderr', subprocess.STDOUT)
+    return subprocess.check_output(cmd, **kwargs).decode()
```

Comment:
Let's specify an encoding. `latin1` is safe for any incoming bytestring and will keep most text as-is.